src/js/index.js

```diff
@@ -1,21 +1,57 @@
 import {tryRegisterSW} from './swRegistration.js';
-import {ServicesTab, TabContentContainer} from './components/tabs.js';
+import {MyMuniTab, ServicesTab, TabContentContainer} from './components/tabs.js';
 import {ModalPage, Service} from './components/pages.js';
 import {API} from './api.js';
 
 // Call as early as possible to maximise chance of registering reinstallation code
 tryRegisterSW();
 
+/**
+ * Creates and returns the HTML for a linked tab
+ * @param {String} tabTemplate - The HTML of the cloned tab template
+ * @param {*} href - The `href` for the anchor link
+ * @returns The linked tab HTML as a string
+ */
+function createTab(tabTemplate, href) {
+  const linkTab = document.createElement("a");
+  linkTab.href = href;
+  linkTab.classList.add("tab-link__wrap");
+
+  linkTab.append(tabTemplate);
+  
+  return linkTab;
+}
+
 class App {
   constructor() {
     this.api = new API();
+
     const tabContentContainer = new TabContentContainer($(".tab-content"));
-    this.servicesTab = new ServicesTab(this.api, $(".main .tab-link").first(), tabContentContainer);
+    
+    const $mainContainer = $(".main");
+    const tabsContainer = $mainContainer.find(".tab-links__wrap");
+
+    const tabTemplate = $mainContainer.find(".tab-link");
+    
+    const myServicesAnchor = createTab(tabTemplate.clone()[0], "/services/");
+    const myMuniAnchor = createTab(tabTemplate.clone()[0], "/my-municipality/");
+    
+    (tabsContainer).append($(myServicesAnchor));
+    (tabsContainer).append($(myMuniAnchor));
+    
+    this.servicesTab = new ServicesTab(this.api, $(myServicesAnchor), tabContentContainer);
+    this.myMuniTabContent = new MyMuniTab(this.api, $(myMuniAnchor), tabContentContainer);
+
+    // HACK TO HIDE DEFAULT ADDED FIRST TAB
+    $mainContainer.find(".tab-link__wrap").first().hide();
```

Comment:
Looks like webflow adds a default first tab. As it will not be an actual link like we need, I added this hack to hide it for now.

Reply:
thanks - you could totally delete this one. I like the consistency of us creating all tabs